lib/controllers/services.ts

```diff
@@ -56,9 +64,23 @@ type IGetServiceHandler = (
   auth: IAzureApiAuthorization,
   clientIp: ClientIp,
   userAttributes: IAzureUserAttributes,
-  serviceId: NonEmptyString
+  serviceId: ServiceId
 ) => Promise<IGetServiceHandlerRet>;
 
+type IGetSenderServicesHandlerRet =
+  | IResponseSuccessJsonIterator<{
+      readonly serviceId: ServiceId;
+      readonly fiscalCode: FiscalCode;
```

Comment:
is this the service fiscal code? (in that case, why it's needed by the client?)

Reply:
it is not, I'll remove it